modules/ROOT/pages/deployment/wopi/wopi.adoc

```diff
@@ -0,0 +1,103 @@
+= Office Applications Using WOPI
+:toc: right
+:wopi_subdir: /ocis_wopi
+:download-gh-directory-url: https://download-directory.github.io
+:traefik_url: https://traefik.io/traefik/
+:nginx_url: https://www.nginx.com
+:apache_url: https://www.apache.org
+
+:description: Infinite Scale uses the WOPI protocol to integrate office applications. This document describes the general flow and basic settings used.
+
+== Introduction
+
+{description} For more details on possible configuration values see the xref:conf-examples/office-integration.adoc[Office Integration] document at the configuration examples. 
+
+For the time being, the supported office applications are:
+
+* Collabora
+* OnlyOffice
+
+== General Information
+
+Infinite Scale can be configured to use office applications via a browser in an iframe. Depending on the use case and among other things, administrators can:
+
+* provide multiple applications from different vendors in parallel,
+* define which icons are used for which applications,
+* define which app is preferred to open a file.
+
+Configured applications can be selected by users when opening a document via the WebUI. The list of available applications to open a file show an icon and its app name.
+
+Office applications are :
+
+* provided via the xref:{s-path}/app-provider.adoc[App-Provider Service] because they are not able to register themself, see the xref:#wopi-coniguration-examples[docker-copmpose.yml] file. 
+
+* configured for the use with the WebUI via the xref:{s-path}/app-registry.adoc[App-Registry Service]. Here, the graphical representation details like icons, app names, mimetypes, app to open etc is set, see the xref:#wopi-coniguration-examples[app-registry.yaml] file.
+
+== WOPI Flow
+
+The following two sections describe the WOPI integration in an overview and a more detailed image to get a better understanding of which components are involved and which basic settings need to be set. You can also extrapolate how to scale office application servers and how to use various office applications in parallel. Note that only a few basic configuration options are shown in the images below to support the process flow.
+
+=== WOPI Overview
+
+This image gives a brief overview of which components and settings are involved. It also shows where parts of an URL are defined, how an URL is assembled, and which component is using which part of the URL.
+
+image::deployment/wopi/wopi-overview.svg[WOPI Overview Diagram]
+
+=== WOPI in Detail
```

Comment:
Again, to verbose. Please keep the information about the WOPI protocol here limited, as that is not the scope of this document. 
Rather put that into the dev docs.

Reply:
I agree with this. While the detailed diagram is quite interesting, I wonder if we really need that level of detail here. It is also somewhat hard to digest without any further description IMO. I think it would be much better live in the dev docs.

Reply:
fixed, see my comment below